web/profiles/idea/modules/openideal_idea/openideal_idea.module

```diff
@@ -43,4 +43,19 @@ function openideal_idea_form_alter(&$form, FormStateInterface $form_state, $form
     // Set default value for challenge form element of the idea content type.
     $form['field_challenge']['widget'][0]['target_id']['#default_value'] = $default_challenge;
   }
+
+  $form['actions']['submit']['#submit'][] = '_openideal_idea_form_submit';
+}
+
+/**
+ * Custom submit handler for idea create/edit form.
+ */
+function _openideal_idea_form_submit($form, FormStateInterface $form_state) {
```

Comment:
Wouldn't it be better to have this logic in `hook_ENTITY_TYPE_presave()`?

Reply:
Yes, it makes sense, I have updated PR.